src/resources/js/tribe-events.js

```diff
@@ -865,11 +865,11 @@ Date.prototype.format = function( mask, utc ) {
 				$tribe_bar.tribe_clear_form();
 			}
 
-			params = tribe_ev.fn.parse_string( params );
+			params = tribe_ev.fn.parse_string( decodeURIComponent( params.replace( /\+/g, '%20' ) ) );
```

Comment:
I still believe that this should be done within the `$.each()` on `value` but only when needed (text/hidden fields).